.prettierrc-nunjucks

```diff
@@ -0,0 +1,9 @@
+{
+  "singleQuote": true,
+  "semi": false,
+  "trailingComma": "none",
+  "arrowParens": "avoid",
+  "printWidth": 120,
+  "proseWrap": "preserve",
+  "tabWidth": 1
+}
```

Comment:
Sure, i can explain :D To format Nunjucks templates with Prettier, I kept getting a "couldn't resolve parser 'nunjucks'," error when adding .njk extension to the 'format' command, and after googling, it's because Prettier doesn't have built-in support for parsing Nunjucks templates we would need an external parser that can understand the Nunjucks syntax, there isn't actually a nunjucks parser package existing (although i kept being directed to one, that package is no longer onthe npm register)

So step 2 - was to parse njk files as html
`"prettier": "prettier --write \"**/*.{js,jsx,ts,tsx,md}\" && **prettier --write \"**/*.njk\" --parser html**"`

but, that kept formatting the nunjucks files to be minified (we want to preserve line wrapping and space between paragraphs) because of the syntax, it was very hard to read! 

step 3, was was to create a bespoke command just for nunjucks files, and ignore it from the original prettier command, i looked for best practice for the nunjucks files and now it has its own prettier set up